src/service/service.js

```diff
@@ -0,0 +1,10 @@
+import { WORDS_COUNT } from '../constants/constants';
+import { getRandomIndexInRange } from '../helpers';
+import { words } from '../data/data';
+
+
+const index = getRandomIndexInRange(WORDS_COUNT);
```

Comment:
attenzione! this will be run only once in whole life of the app
so it wouldn't make sense to restart the game without refreshing the page (ctrl+R)

getting random words should happen every time someone calls `getCorrectWords` (once per game)

Reply:
I want the project to go forward so maybe fix the above error in this PR but address the other two issues in a separate smaller PR right after the current one.

Reply:
I had modified `handleLetterClick` before I read it, so I'm pushing it, ooops